sample_apps/simple_app_under_sev/sev_policy.json

```diff
@@ -19,7 +19,7 @@
           "comparator": "eq",
           "type": "string",
           "name": "smt",
-          "value": "no"
+          "value": "yes"
```

Comment:
Let's not commit these changes. The default ones here are for the simulator and should be preserved.

Reply:
Done. I've withdrawn the entire file and all the changes. Will re-test.

Reply:
Re-tested. Script still runs cleanly.